### ISSUE TITLE
Add aria-label to button

### ISSUE THREAD

**Issue by davidsneighbour** [luwes/lite-vimeo-embed#12]
https://github.com/luwes/lite-vimeo-embed/blob/62339ecec7e1417101bfd68176ad4d0dd1fe520d/src/lite-vimeo-embed.js#L44

The play button throws errors in several accessibility tools because a button should have a label for screenreaders. Please add something like `aria-label="Play video"` or even make it configurable?
